lms/static/images/arrow-left.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 16 16" aria-hidden="true" focusable="false" class="Icon Icon--arrow-left"><g fill-rule="evenodd"><rect fill="none" stroke="none" x="0" y="0" width="24" height="24"></rect><path fill="none" stroke="white" stroke-linecap="square" stroke-linejoin="miter" stroke-width="2" d="M7 12L3 8l4-4M4 8h9-9z"></path></g></svg>
```

Comment:
This is from the style guide 
https://hypothesis-pattern-library.netlify.com/

**I did change the color

Reply:
https://github.com/hypothesis/lms/issues/1014